.benchmark/hello.yml

```diff
@@ -2,7 +2,7 @@ config:
   target: 'http://localhost:3004'
   phases:
     - duration: 60
-      arrivalRate: 300
+      arrivalRate: 200
```

Comment:
What's the reason for change of rate? Does this relate to the client.setex line in server/index.js?